ekf_estimator/src/ekf_estimator.cpp

```diff
@@ -1,22 +1,42 @@
 #include "ekf_estimator/ekf_estimator.h"
 
+
+const int numofStates = 6;
+const int num_feet_ = 4;
 EKFEstimator::EKFEstimator(ros::NodeHandle nh) {
   nh_ = nh;
 
   // Load rosparams from parameter server
   std::string joint_encoder_topic, imu_topic, contact_topic, state_estimate_topic;
-  nh.param<std::string>("topics/joint_encoder", joint_encoder_topic, "/joint_encoder");
-  nh.param<std::string>("topics/imu", imu_topic, "/imu");
+  nh.param<std::string>("topics/mcu/state/jointURDF", joint_encoder_topic, "/mcu/state/jointURDF");
+  nh.param<std::string>("topics/mcu/state/imu", imu_topic, "/mcu/state/imu");
```

Comment:
Is there a reason for this change? I think we want to load them from the topics/joint_encoder and topics/imu parameters (although this won't matter once this is all implemented within RobotDriver).

Reply:
Yes. Currently, I can only get the imu and encoder data from the bag file and they are published to "topics/mcu/state/jointURDF" and "topics/mcu/state/imu". But yeah, it won't matter once this is implemented within RobotDriver.